src/deluge/gui/views/session_view.cpp

```diff
@@ -2352,29 +2352,32 @@ void SessionView::displayPotentialTempoChange(UI* ui) {
 /// display number of bars or quarter notes remaining until a launch event
 int32_t SessionView::displayLoopsRemainingPopup(bool ephemeral) {
 	int32_t sixteenthNotesRemaining = session.getNumSixteenthNotesRemainingTilLaunch();
-	if (sixteenthNotesRemaining > 0) {
-		DEF_STACK_STRING_BUF(popupMsg, 40);
-		if (sixteenthNotesRemaining > 16) {
-			int32_t barsRemaining = ((sixteenthNotesRemaining - 1) / 16) + 1;
-			if (display->haveOLED()) {
-				popupMsg.append("Bars Remaining: ");
+	// don't show pop-up if you're pressing a clip in song view
+	if (currentUIMode != UI_MODE_CLIP_PRESSED_IN_SONG_VIEW) {
```

Comment:
Can we check to be in UI mode none? This seems likely to cause bugs later

Reply:
> Can we check to be in UI mode none? This seems likely to cause bugs later

Done